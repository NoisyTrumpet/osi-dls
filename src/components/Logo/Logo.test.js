import React from "react";
import { render } from "@testing-library/react";
// import {
//   toBeInTheDocument,
//   toHaveClass,
//   toHaveAttribute,
// } from '@testing-library/jest-dom/matchers';
import { Logo } from "./";

// expect.extend({
//   toBeInTheDocument,
//   toHaveClass,
//   toHaveAttribute,
// });

describe("Logo", () => {
  test("renders Logo component", () => {
    const { container } = render(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
