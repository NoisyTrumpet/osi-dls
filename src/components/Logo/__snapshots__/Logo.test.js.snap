// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo component 1`] = `
<div
  class="logo-wrapper"
/>
`;
